jobs/build/tarball-sources/Jenkinsfile

```
@@ -0,0 +1,165 @@
+                        name: 'ADVISORY',
+                        description: '(REQUIRED) Image release advisory number.',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: ""
+                    ],
+                    [
+                        name: 'RCM_GUEST',
+                        description: 'Details of RCM GUEST',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: "ocp-build@rcm-guest.app.eng.bos.redhat.com:/mnt/redhat/staging-cds/"
+                    ],
+                    [
+                        name: 'MAIL_LIST_SUCCESS',
+                        description: 'Success Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'release-engineering@redhat.com',
+                        ].join(',')
+                    ],
+                    [
+                        name: 'MAIL_LIST_FAILURE',
+                        description: 'Failure Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'aos-art-automation+failed-release@redhat.com'
+                        ].join(',')
+                    ],
+                    commonlib.mockParam(),
+                ]
+            ],
+        ]
+    )
+
+    commonlib.checkMock()
+    buildlib.initialize()
+    buildlib.cleanWorkdir(env.WORKSPACE)
+
+    try {
+        sshagent(['openshift-bot']) {
+            advisory = params.ADVISORY ? Integer.parseInt(params.ADVISORY.toString()) : 0
+            if (advisory == 0) {
+                error "Need to provide an ADVISORY ID"
+            }
+            def components = params.COMPONENTS.replaceAll(',', ' ').split(' ')
+            def filename = "tarball-source-output.txt"
+            def outdir = "/mnt/nfs/home/jenkins/container-sources/"
+            def rcm_guest = params.RCM_GUEST.toString()
+            currentBuild.description = "tarball: ${components}"
+
+            stage("elliott tarball-source") {
+                component_args = ""
+                components.each {
+                    component_args += "--component ${it}"
+                }
+                buildlib.elliott """
+                    tarball-sources create
+                    --out-dir=${outdir}
+                    --force
+                    ${component_args}
+                    ${advisory} > ${filename}
+                """
+            }
+
+            stage("rsync") {
+                cmd = "rsync -avz --chmod=go+rX ${outdir} ${rcm_guest}"
+                commonlib.shell(
+                    script: cmd,
+                    returnAll: true
+                )
+            }
+
+            stage("file ticket to RCM") {
+                // use awk to print out the sigle line cotains ${advisory}
```

```suggestion
                // use awk to print out the lines that contain ${advisory}
```

We only have one now but it seems like we could theoretically have more.